src/blk/kernel/KernelDevice.cc

```diff
@@ -541,6 +541,27 @@ void KernelDevice::_aio_stop()
   if (aio) {
     dout(10) << __func__ << dendl;
     aio_stop = true;
+    IOContext wakeup_ctx(cct, nullptr, false);
+    wakeup_ctx.num_running++;
+
+    std::list<aio_t> batch;
+    bool use_ioring = cct->_conf.get_val<bool>("bdev_ioring");
+    if (use_ioring && ioring_queue_t::supported()) {
+       aio_t aio_op(&wakeup_ctx, -1);
+       aio_op.iocb.aio_lio_opcode = IO_CMD_NOOP;
+       batch.push_back(aio_op);
```

Comment:
Possibly if we set fd:
`aio_op.iocb.aio_fildes = fd_directs[WRITE_LIFE_NOT_SET];`
we could revert to simpler function `init_sqe()`